test/src/unit-tile-metadata-generator.cc

```diff
@@ -348,7 +348,7 @@ TEST_CASE(
   bool empty_tile = test == "empty tile";
 
   uint64_t max_string_size = 100;
-  uint64_t num_strings = 2000;
+  int num_strings = 2000;
```

Comment:
I wasn't able to reproduce it or figure out why but this line of code started producing warnings in the windows build which failed the CI checks.
```
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30133\include\xutility(135,1): warning C4267: 'initializing': conversion from 'size_t' to '_Ty', possible loss of data [D:\a\tdbbd\tiledb\test\tiledb_unit.vcxproj]
          with
          [
              _Ty=int
          ] (compiling source file D:\a\TileDB\TileDB\test\src\unit-tile-metadata.cc)
```
The operation `rand() % num_strings` produces a `uint64_t` which is truncated to `int` when it is pushed onto `values`.  Since `num_strings` is 2000 this probably shouldn't have produced a warning with any compiler, but it was doing so in the `msvc` job.